src/prototype_room_my.js

```diff
@@ -284,7 +284,31 @@ Room.prototype.getHarvesterAmount = function() {
   return amount;
 };
 
+Room.prototype.visualizz = function() {
+
+  let color = function(coeff) {
+    return '#' + (Math.floor(255 - coeff * 239)).toString(16) + (Math.floor(coeff * 239 + 16)).toString(16) + '00'
+  }
+  this.visual.text(`Energie Average : ${Math.floor(this.memory.energyAvailable)}`, 5,0.5, {
+    color: color(Math.min(this.memory.energyAvailable, 5000) / 5000),
+    font: 1
+  });
+  this.visual.text(`| Storage Energy : ${this.storage && this.storage.store.energy}`, 17,0.5, {
```

Comment:
Don't hardcode to .energy - follow the constants, because why not?
`this.storage.store.energy` => `this.storage.store[RESOURCE_ENERGY]`
Along with this, change `Storage Energy` to `Stored Energy`

Reply:
alright, thx